packages/big-design/src/components/Worksheet/editors/CheckboxEditor/CheckboxEditor.tsx

```diff
@@ -8,20 +8,35 @@ import { CheckboxWrapper } from './styled';
 
 export interface CheckboxEditorProps<Item> {
   cell: Cell<Item>;
+  isEditing: boolean;
+  onBlur(): void;
   onChange(value: unknown): void;
 }
 
-const InternalCheckboxEditor = <T extends WorksheetItem>({ cell, onChange }: CheckboxEditorProps<T>) => {
+const InternalCheckboxEditor = <T extends WorksheetItem>({
+  cell,
+  isEditing,
+  onBlur,
+  onChange,
+}: CheckboxEditorProps<T>) => {
+  useEffect(() => {
+    if (isEditing) {
+      // For checkbox we want to change the value when editing is called
+      onChange(!cell.value);
+    }
+  }, [cell.value, isEditing, onChange]);
+
   const handleChange = () => {
```

Comment:
Just curious: for the checkbox, what's the difference in `isEditing` and `onChange` - are they effectively the same? Editing and "changing" the value?

Reply:
They are the same, except `onChange` is called when the checkbox is mouse clicked and the `isEditing` state changes when a user presses enter or space when the cell is selected. I will add some comments and change things to make it more clear.